javascript/0187-repeated-dna-sequences.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * https://leetcode.com/problems/repeated-dna-sequences/
+ * Hashing
+ * s = the number of letters in the sequance. In our case it's 10. so the time complexity would be 10*n which boils down to n.
+ * Time O(s*n) | Space O(n)
+ * @param {string} s
+ * @return {string[]}
+ */
+
+var findRepeatedDnaSequences = function(s) {
+
+    const sequanceStack = new Set();
```

Comment:
The variable names are misleading

```js
const sequenceSet = new Set();
const resultSet = new Set();

return [ ...resultSet ];
```

Reply:
Oh, my bad. Sorry. I updated the variables.